test/run-tests

```diff
@@ -280,7 +291,16 @@ class checkout_repository:  # pylint: disable=invalid-name
             self.refspec,
             env={"GIT_TERMINAL_PROMPT": "0"},
         )
-        run("git", "-C", self.dir.name, "checkout", "--quiet", "FETCH_HEAD")
+        run(
+            "git",
+            "--git-dir",
+            self.dir.name + "/.git",
+            "--work-tree",
+            self.dir.name,
+            "checkout",
+            "--quiet",
+            "FETCH_HEAD",
```

Comment:
These changes were made so that I could use `git` 1.8 on centos7 - the `-C` argument was added in git 2.0.

Reply:
Note that there is a newer git available in Software Collections.

Reply:
https://wiki.centos.org/AdditionalResources/Repositories/SCL . Note that https://wiki.centos.org/SpecialInterestGroup/SCLo/CollectionsList is obsolete, I see rh-git218-git in Collections.

Reply:
I will see if I can use scl git 2 and revert this change

Reply:
fixed in https://github.com/linux-system-roles/test-harness/pull/87/commits/f6a2c4331b324de0f736ab11f9ce451f459f9468

Also note - for some reason, this causes the use of py36 which made another change necessary - https://github.com/linux-system-roles/test-harness/pull/87/commits/548311ce89fb1ea8d17fba8063681e336b7a1c18